spec/unit/inspect_task_spec.rb

```diff
@@ -94,7 +111,15 @@ def inspect(_system, description, _filter, _options = nil)
       Filter.new
     )
 
-    expect(description.foo).to eql(SimpleInspectTaskScope.new("bar" => "baz"))
+    expected = SimpleInspectTaskScope.new(
+      files: SimpleInspectTaskList.new([
+          Machinery::Object.new(name: "foo"),
```

Comment:
Use 2 spaces for indentation in an array, relative to the start of the line where the left bracket is.